tox.ini

```diff
@@ -6,16 +6,16 @@ skipsdist = True
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
-commands = py.test --ignore=opaque_keys/edx/django {posargs}
+commands = py.test --disable-pytest-warnings --ignore=opaque_keys/edx/django {posargs}
```

Comment:
I understand the problem with the log length, but hiding these warnings concerns me. I think we need a high-priority ticket to fix enough warnings that we can turn these back on ASAP.

Reply:
Agreed